public_dropin_gpu_environments/nemollm_inference_ms/start_server.sh

```diff
@@ -32,25 +27,26 @@ if [ "${ENABLE_CUSTOM_MODEL_RUNTIME_ENV_DUMP}" = 1 ]; then
     /opt/nvidia/nvidia_entrypoint.sh /bin/true
 fi
 
-source /home/nemo/dr/bin/activate
-echo
-echo "Starting DRUM server in background..."
-echo
-nohup drum server --with-triton-server "$@" &
-deactivate
 
 echo
 echo "Starting NeMo Inference Microservice..."
 echo
-export MODEL_DIR="${CODE_DIR}/model-store/"
-
 export NEMO_PORT=9998
 export OPENAI_PORT=9999
 export HEALTH_PORT=8081
+export MODEL_NAME=generic-llm
+export MODEL_DIR="${CODE_DIR}/model-store/"
 
-exec nemollm_inference_ms --model $MODEL_NAME \
+nohup nemollm_inference_ms --model $MODEL_NAME \
     --log_level=info \
     --health_port=$HEALTH_PORT \
     --openai_port=$OPENAI_PORT \
     --nemo_port=$NEMO_PORT \
-    --num_gpus=$(nvidia-smi -L | wc -l)
+    --num_gpus=$GPU_COUNT &
+
+
+echo
+echo "Starting DRUM server..."
+echo
+source /home/nemo/dr/bin/activate
+exec drum server --with-nemo-server "$@"
```

Comment:
Perhaps I'll need to start NIM from instide of the DRUM, after the model is loaded. I'm experimenting with that, but it will be a separate PR.

Reply:
Yeah, I forgot about that because I found too that it seems NIM just had one shot to start and have models there. Be careful of starting from DRUM though because I installed drum and stuff into dedicated venv due to package conflicts so from drum, you will want to deactivate venv before calling command (I think subprocess module supports pre-hook)